stable/redis-ha/templates/redis-master-appliance-vpc.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: redis-master-appliance-vpc
```

Comment:
It's customary in the official charts to create all resources with the fullname variable in front of it.

Reply:
Switched to dynamic names.